plugins/build-packages.js

```diff
@@ -0,0 +1,21 @@
+const { includes, excludes } = require('./configs.json');
+const { execSync } = require('child_process');
+
+let scope, ignore;
+
+if (includes && includes.length > 0) {
+  scope = includes.map((e) => `--scope @tooljet-plugins/${e}`).join(' ');
+}
+
+if (!scope && excludes && excludes.length > 0) {
+  ignore = excludes.map((e) => `--ignore @tooljet-plugins/${e}`).join(' ');
+}
+
+const buildCommand = `lerna ${
+  scope
+    ? `--scope @tooljet-plugins/common --scope @tooljet-plugins/restapi ${scope} `
+    : ignore
```

Comment:
Does it make sense to only have includes by default since we can't use both? 🤔 
That'd mean we only maintain a whitelist by default and all the packages will be listed in _configs.json_.
If the user only wants specific packages, can simply comment others out?